serverless.yml

```diff
@@ -1,12 +1,16 @@
 # For full config options, check the docs:
 #    docs.serverless.com
 
-service: projectworks-leave-notifications
+# the service name is used as a prefix for infrastructure created via
+# CloudFormation. Some infrastructure has limits on name length so don't make
+# the service name too long
+service: projectworks-leave-notify
```

Comment:
Had to shorten the name because cloudformation was failing when deploying to the `staging` stage ("staging" being longer than "dev" or "prod")